processor/app.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from parser_report import ExcelParser
+from loggers.logger import Logger
+import os
+from flask import jsonify
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    parser = ExcelParser(logger=Logger)
+    parsed = parser.parse_file(file_path=os.path.join(os.getcwd(), "assets", "513026484_gs.xlsx"))
+    return jsonify({'results': parsed})
+
+
+if __name__ == '__main__':
+    app.run(debug=True, host='0.0.0.0')
```

Comment:
It's broken:
```bash
processor_1  | Traceback (most recent call last):
processor_1  |   File "app.py", line 18, in <module>
processor_1  |     app.run(debug=(os.environ['ENV'] == 'dev' or False), host='0.0.0.0', port=80)
processor_1  |   File "/usr/local/lib/python3.7/os.py", line 678, in __getitem__
processor_1  |     raise KeyError(key) from None
processor_1  | KeyError: 'ENV'
``` 

I'll try to fix this one.

Reply:
sry my bad, we should've used `.get()`.

this won't throw `KeyError` since it has a default of `None` if the key doesn't exist:
```python
# instead of os.environ['ENV'] use os.environ.get(...)
(os.environ.get('ENV') == 'dev' or False)
```

Reply:
@RoySegall ^